perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuditMessagesManagerBlImpl.java

```diff
@@ -69,6 +74,23 @@ public Paginated<AuditMessage> getMessagesPage(PerunSession perunSession, Messag
 		return getAuditMessagesManagerImpl().getMessagesPage(perunSession, query);
 	}
 
+	@Override
+	public List<String> findAllPossibleEvents(PerunSession sess) {
```

Comment:
I tried the following code, and it seems to be working on our devel:

	public List<String> findAllPossibleEvents(PerunSession sess) {
			Reflections reflections = new Reflections(AuditEvent.class);
			return (reflections.getSubTypesOf(AuditEvent.class).stream().map(Class::getSimpleName).collect(Collectors.toList()));
	}